baseplate/server/__init__.py

```diff
@@ -109,6 +111,18 @@ def parse_args(args: Sequence[str]) -> argparse.Namespace:
     return parser.parse_args(args)
 
 
+class EnvironmentInterpolation(configparser.Interpolation):
+    def before_get(
```

Comment:
cribbing off typeshed's hints for this API https://github.com/python/typeshed/blob/master/stdlib/configparser.pyi#L41